orcid-api-common/src/main/java/org/orcid/api/common/jaxb/OrcidValidationJaxbContextResolver.java

```diff
@@ -295,6 +323,11 @@ private JAXBContext getJAXBContext(String apiVersion) {
                         jaxbContext_2_1 = JAXBContext.newInstance(SCHEMA_FILENAME_PREFIX_BY_CLASS_V2_1.keySet().toArray(new Class[SCHEMA_FILENAME_PREFIX_BY_CLASS_V2_1.size()]));
                     }
                     return jaxbContext_2_1;
+                } else if(apiVersion.equals("3.0_dev1")) {
+                    if(jaxbContext_3_0_dev1 == null) {
+                        jaxbContext_3_0_dev1 = JAXBContext.newInstance(SCHEMA_FILENAME_PREFIX_BY_CLASS_V3_0_DEV1.keySet().toArray(new Class[SCHEMA_FILENAME_PREFIX_BY_CLASS_V2_1.size()]));
```

Comment:
Hey @bobcaprice,

Please change SCHEMA_FILENAME_PREFIX_BY_CLASS_V2_1.size() to SCHEMA_FILENAME_PREFIX_BY_CLASS_V3_0_DEV1.size()

Thanks